sdk/storage/storage-queue/src/utils/constants.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export const SDK_VERSION: string = "12.12.0";
+export const SDK_VERSION: string = "12.20.0";
 export const SERVICE_VERSION: string = "2021-10-04";
```

Comment:
is it really that old?

Reply:
No, I just had pushed all of the storage packages in main to 20 to not conflict with the stable branch until we were ready to release out of main again. We can bump these back down as needed. There's some catch up work before we can wind down stable that Jeremy is tracking.

Reply:
sorry I wasn't clear, I meant to refer to L5 that has the service version.

Reply:
oh! Yeah, the queue service hasn't really been changed in a long time, it's mostly on life support is my understanding.